packages/create-pantheon-decoupled-kit/src/templates/tailwindless-next/components/layout.module.css

```diff
@@ -0,0 +1,26 @@
+.layout {
+  display: flex;
+  flex-direction: column;
+  min-height: 100dvh;
+  min-width: 100dvw;
+}
+
+.layout > main {
+  margin-bottom: auto;
+}
+
+.footerCopy {
+  margin: var(--x-auto);
+}
+
+.footerCopy > [href] {
```

Comment:
Targeting the href attribute for something like this is a little unusual. Any reason this isn't `a`? Same comment goes for similar selectors throughout.

Reply:
No good reason, replacing with `a` :)